packages/core/src/app.ts

```diff
@@ -1195,16 +1206,24 @@ export class AppBuilder<TreeNames> {
 }
 
 const entryTemplate = compile(`
-import { importSync as i } from '@embroider/macros';
+import { importSync as i, macroCondition, getGlobalConfig } from '@embroider/macros';
 let w = window;
 let d = w.define;
 
+{{#if styles}}
+  if (macroCondition(getGlobalConfig().fastboot?.isRunning)) {}
```

Comment:
@ef4 this might be a uses case for having a falsy macro or do you have a suggestion here?

Reply:
(we discussed this out of band, but I'm commenting for other readers)

You can't say `if (!macroCondition(whatever))` but you can say `if (macroCondition(!whatever))`.

We could choose to make the other case work too. It would mean that the presence of `macroCondition` *anywhere* in the predicate requires the whole predicate to be statically determined (and generate a build error if it's not).